extensions/data-workspace/src/common/constants.ts

```diff
@@ -23,6 +23,7 @@ export const DoNotShowAgain = localize('dataworkspace.doNotShowAgain', "Do not s
 export const ProjectsFailedToLoad = localize('dataworkspace.projectsFailedToLoad', "Some projects failed to load. Please open console for more information");
 export const fileDoesNotExist = (name: string): string => { return localize('fileDoesNotExist', "File '{0}' doesn't exist", name); };
 export const projectNameNull = localize('projectNameNull', "Project name is null");
+export const noPreviousData = (tableName: string): string => { return localize('noPreviousData', "No previous {0} to display", tableName); };
```

Comment:
Should we add "please run build/deploy to see the results" 


+ @dzsquared for any suggestions

Reply:
@dzsquared The only information that we have here is "Deployments" and "Builds" for Sql Projects. If something can be orchestrated around these strings, that would work.

Reply:
"Prior <deployments/builds> will appear here, please run to see the results."

Reply:
@SakshiS-harma for stuff like this, you should just have 2 localizable strings.

Reply:
@alanrenmsft for my undertsanding - why two if the string is same with only {builds/deployments} word being different?

Reply:
@udeeshagautam speaking of localized strings, the current approach requires 3 strings. builds + deployments + this string. this is what I would do, not really a big deal.

Reply:
oh i was thinking 1 vs 2 - didnt realize 3 :/ hmm.

Reply:
@alanrenmsft "Builds", "Deployments" strings are already localized in Sql Db Projects extension.
https://github.com/microsoft/azuredatastudio/blob/b774f09b6cb6fb3e219972354907d5d2b91a6567/extensions/sql-database-projects/src/common/constants.ts#L44

Reply:
Updated the text.